packages/integration-tests/index.js

```diff
@@ -95,20 +92,38 @@ async function getExecutionStatus(executionArn, retryOptions) {
  */
 async function waitForCompletedExecution(executionArn, timeout = 600) {
   let executionStatus;
-
+  let iteration = 0;
   const stopTime = Date.now() + (timeout * 1000);
-
   /* eslint-disable no-await-in-loop */
   do {
-    executionStatus = await getExecutionStatus(executionArn);
-    if (executionStatus === 'RUNNING') await sleep(5000);
+    try {
+      executionStatus = await getExecutionStatus(executionArn);
+    }
+    catch (err) {
+      if (!(err.code === 'ExecutionDoesNotExist') || iteration > 12) {
+        console.log(`waitForCompletedExecution failed: ${err.code}`);
+        throw err;
+      }
+      console.log("Execution does not exist... assuming it's still starting up.");
+      executionStatus = 'RUNNING';
+    }
+    if (executionStatus === 'RUNNING') {
+      console.log("Execution started");
```

Comment:
This is super nitpicky so feel free to ignore, but I got a little confused in the build output and I think 'Execution still running' would make more sense. But feel free to leave it or change it in a future PR. I also think we don't need to output the heartbeat since we're doing a console.log here.

Reply:
Not nit-picky at all.     I'll remove this line and change the heartbeat output to something better.

Reply:
9fbef214